conda_forge_tick/config_schema.py

```diff
@@ -246,6 +246,13 @@ class BotConfig(BaseModel):
         description="Automatically remake untouched bot PRs with conflicts.",
     )
 
+    pr_refresh_age_days: Optional[float] = Field(
+        default=7.0,
+        description="Number of days after which to refresh PR cache for 'clean' PRs "
+        "to detect potential conflicts. Works around GitHub API bug #5150 where "
+        "Last-Modified caching can hide merge conflicts. Set to 0 to always refresh.",
+    )
```

Comment:
This is in the wrong file. Internal settings for the bot go in this file: https://github.com/regro/cf-scripts/blob/main/conda_forge_tick/settings.py

Reply:
Yep, that make a lot more sens.